src/scss/components/contact.scss

```diff
@@ -0,0 +1,43 @@
+.contactUs {
+  @extend .container;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-left: 19px;
+  padding-right: 19px;
+
+  &__title {
+    display: flex;
+    font-family: $hkgrotesk-bold;
+    font-size: $mobile-title;
+    padding-bottom: 20px;
```

Comment:
Use `margin-bottom` instead of `padding-bottom` to add spacing between elements. Remember, the `padding` is internal, and the `margin` is external to the element. Please apply this where needed.